openstax-poland/adaptarr-front
# Scroll to an element on xref ctrl + click

https://github.com/openstax-poland/adaptarr-backlog/issues/47

### Testing:
1. Go to a draft
2. Hold Ctrl and Click on Xref to a figure, note, etc. (ex. `Figure 1` / `Note 1`)
3. If the target element is in the current document then the screen should be scrolled to the top of target element
4. Url should change
5. Refresh - you should be scrolled to en element after everything is loaded
6. If you ctrl + click on Xref which lead to another document then this document should be opened in new tab and scrolling behaviour should stay the same.
